docs/index.html

```diff
@@ -392,22 +392,27 @@ <h2>Data Model</h2>
       </p>
 
       <section>
-        <h3 id="Profile">Profile</h3>
+        <h3 id="Collection">Collection</h3>
 
-        <p>A profile is a named bucket for grouping wallet content.</p>
+        <p>A collection is a named bucket for grouping wallet content.</p>
         <p>
-          A profile MAY be used to group all correlatable wallet items together.
+          A collection MAY be used to group all correlatable wallet items together.
         </p>
         <p>
-          A profile MAY be used to manage separate external personal, or
-          organizational identities and their associated data.
+          A collection MAY be used to aggregate personal or
+          organizational identities and their associated data. 
+        </p>
+        <p>
+          Collection membership is not exclusive; e.g. a credential representing
```

Comment:
```suggestion
          Collection membership is not exclusive; e.g., a credential representing
```

Reply:
I promise that one day it will sink it. Believe me, it's helping sometimes (catching myself every other time) :)